src/Network/NWBrowser.cs

```diff
@@ -146,6 +197,11 @@ public void SetChangesHandler (Action<NWBrowseResult, NWBrowseResult> handler)
 			}
 		}	
 
+		// let to not change the API, but would be nice to remove it in the following releases.
+#if !XAMCORE_4_0
+		public void SetChangesHandler (Action<NWBrowseResult, NWBrowseResult> handler) => IndividualChangesDelegate = handler;
```

Comment:
add an `[Obsolete]` with the alternative
Goal is that people building without (obsolete) warnings will have an easy migration to newer profiles